assets/component-cart-items.css

```diff
@@ -20,6 +21,7 @@
 }
 
 .cart-item__image {
+  height: auto;
   max-width: 100%;
 }
```

Comment:
How should elements be aligned compared to the image ? It seems like it might need to be adjusted: 
![](https://screenshot.click/21-08-78nkl-uhrkm.png)

Reply:
Good point, it should be centered vertically.

Let me know your thoughts

[![alt](https://screenshot.click/19-22-4y0n1-l6lrx.png)](https://screenshot.click/19-22-4y0n1-l6lrx.png)